.pre-commit-config.yaml

```diff
@@ -75,6 +75,7 @@ repos:
                 entry: bash cpp/scripts/run-cmake-format.sh cmake-format
                 language: python
                 types: [cmake]
+                exclude: ^.*thirdparty.*$
```

Comment:
Why exclude the thirdparty directory? That's a bunch of our own .cmake files, so I would think we would want to format them.

Reply:
That's my mistake, I assumed that since these were for finding third party libraries we were also using the CMake configs that those projects provided to make them discoverable. I didn't parse the contents of those files to confirm. I'll add that now.